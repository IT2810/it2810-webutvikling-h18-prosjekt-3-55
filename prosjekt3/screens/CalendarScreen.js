import React from 'react';
import { ScrollView, StyleSheet, Text } from 'react-native';
import { Calendar, CalendarList, Agenda } from 'react-native-calendars';
import {store} from "../components/Storage";

export default class CalendarScreen extends React.Component {
  static navigationOptions = {
    title: 'Calendar',
  };

  render() {
    return (
        <Calendar
            onDayPress={(day)=>{store("dates",day.dateString)}}
        />
    );
  }
}
const styles = StyleSheet.create({
    tran: {
        fontSize: 17,
        color: 'rgba(96,100,109, 1)',
        lineHeight: 24,
        textAlign: 'center',
    }
});
