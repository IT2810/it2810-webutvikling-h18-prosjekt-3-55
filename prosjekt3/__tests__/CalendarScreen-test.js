import 'react-native';
import React from 'react';
import CalendarScreen from '../screens/CalendarScreen';

import renderer from 'react-test-renderer';

test('snapshot linksscreen', () => {
    const tree = renderer.create(<CalendarScreen />).toJSON();
    expect(tree).toMatchSnapshot();
});