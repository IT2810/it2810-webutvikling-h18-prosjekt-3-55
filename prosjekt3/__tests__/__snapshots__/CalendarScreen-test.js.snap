// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot linksscreen 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Object {
      "color": "rgba(96,100,109, 1)",
      "fontSize": 17,
      "lineHeight": 24,
      "textAlign": "center",
    }
  }
>
  hallo
</Text>
`;
